.internal-ci/docker/entrypoints/fogingest.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Copyright (c) 2018-2022 The MobileCoin Foundation
+
+set -eo pipefail
+
+data="/fog-data"
+
+if [[ -n "${MC_LEDGER_DB_URL}" ]]
+then
+    ### CBB: these should use ENV vars for configuration.
+    #   Need to fix .internal-ci/helm/mc-core-common-config/templates/mobilecoind-supervisord-mobilecoind-configmap.yaml
+    echo "MC_LEDGER_DB_URL set, restoring ${data}/ledger/data.mdb from backup"
+    if [[ -f "${data}/ledger/data.mdb" ]]
+    then
+        echo "Found existing ledger database, skipping download"
+    else
+        echo "Downloading ledger data.mdb"
+        curl -L "${MC_LEDGER_DB_URL}" -o "${data}/ledger/data.mdb"
```

Comment:
What happens if this `curl` fails?  Does the entrypoint crash?  I guess a way to test it is to set a known invalid `${MC_LEDGER_DB_URL}` and confirm what happens.

Reply:
If the curl fails the entrypoint exits error and the pod will error.  I think this is the behavior we want.  At this point we would want a human to fix the url var or remove the value.

If the pod already has a database, the curl doesn't happen.  So the error condition should only happen on a fresh new deployment without an existing .mdb file if the URL var is set.